arex-instrumentation-api/src/main/java/io/arex/inst/runtime/util/DatabaseUtils.java

```diff
@@ -42,35 +44,21 @@ public static String regenerateOperationName(String dbName, String operationName
         if (CollectionUtil.isEmpty(operationNames)) {
             return operationName;
         }
+        // ensure that the order of multiple SQL statements is the same
+        operationNames.sort(String::compareTo);
         return StringUtil.join(operationNames, ";");
     }
 
-    /**
-     * compatible with the old version, if the excludeMockTemplate config not contains '@', it means that not need generate
-     */
-    private static boolean needRegenerate(String dbName) {
-        Map<String, Set<String>> excludeMockTemplate = ContextManager.currentContext().getExcludeMockTemplate();
-        if (excludeMockTemplate == null) {
-            return false;
-        }
-        Set<String> operationSet = excludeMockTemplate.get(dbName);
-        if (CollectionUtil.isEmpty(operationSet)) {
-            return false;
-        }
-        for (String operation : operationSet) {
-            if (operation != null && operation.contains(DELIMITER)) {
-                return true;
-            }
-        }
-        return false;
-    }
-
     private static String regenerateOperationName(TableSchema tableSchema, String originOperationName) {
         return new StringBuilder(100).append(StringUtil.defaultString(tableSchema.getDbName())).append(DELIMITER)
                 .append(StringUtil.defaultString(StringUtil.join(tableSchema.getTableNames(), ","))).append(DELIMITER)
                 .append(StringUtil.defaultString(tableSchema.getAction())).append(DELIMITER)
                 .append(originOperationName)
                 .toString();
     }
+
+    public static boolean disableSqlParse() {
+        return Config.get().getBoolean(ArexConstants.DISABLE_SQL_PARSE, Boolean.getBoolean(ArexConstants.DISABLE_SQL_PARSE));
```

Comment:
ENABLE_SQL_PARSE

Reply:
also default open, can only configure it when it needs to be closed